llvm/test/CodeGen/NVPTX/idioms.ll

```diff
@@ -114,8 +114,8 @@ define %struct.S16 @i32_to_2xi16_shr(i32 noundef %i){
   %h = trunc i32 %h32 to i16
 ; CHECK:      ld.param.u32    %[[R32:r[0-9]+]], [i32_to_2xi16_shr_param_0];
 ; CHECK:      shr.s32         %[[R32H:r[0-9]+]], %[[R32]], 16;
-; CHECK-DAG    mov.b32       {tmp, %rs{{[0-9+]}}}, %[[R32]];
-; CHECK-DAG    mov.b32       {tmp, %rs{{[0-9+]}}}, %[[R32H]];
+; CHECK-DAG:    mov.b32       {tmp, %rs{{[0-9+]}}}, %[[R32]];
+; CHECK-DAG:    mov.b32       {tmp, %rs{{[0-9+]}}}, %[[R32H]];
```

Comment:
Minor nit: instructions in the check strings should be aligned to the same initial column, example:
```suggestion
; CHECK-DAG:  mov.b32       {tmp, %rs{{[0-9+]}}}, %[[R32H]];
```